lfs.c

```diff
@@ -5233,38 +5233,67 @@ static int lfs_fs_gc_(lfs_t *lfs) {
 #endif
 
 #ifndef LFS_READONLY
+static int lfs_fs_rewrite_block_count(lfs_t *lfs, lfs_size_t block_count) {
```

Comment:
nit: Naming is a bit weird, but useful, in littlefs. Underscores indicate namespaces/subsystems for the most part. So this should be `lfs_fs_rewriteblockcount`.

Reply:
Makes sense. This function will be removed anyway following shrink being part of `lfs_fs_grow`.